README.md

```diff
@@ -476,6 +476,37 @@ class Service {
 }
 ```
 
+To resolve conflicts between multiple methods that can be picked for recovery. You can explicitly specify recovery method name. 
+The following example shows how to do so:
+
+```java
+@Service
+class Service {
+
+    @Retryable(recover = "service1Recover", value = RemoteAccessException.class)
+	public void service1(String str1, String str2) {
```

Comment:
Please, consider to not use *tabs* as indents in docs - just *4 spaces* instead.
Otherwise you see how code is not aligned properly

Reply:
Sure. Fixed.